lib/pakyow/views/console/data/datum/_actions.slim

```diff
@@ -1,18 +1,20 @@
+
 label.float-r.datum-settings-trigger for="datum-settings-trigger"
   i.fa.fa-tasks
 
-input.datum-submit type="submit" value="Save" form="datum-form"
+.button-wrapper
```

Comment:
Let's give this a more descriptive class, like `data-button-wrapper`. It'll be easier to distinguish later.